src/ReleaseHistory.md

```diff
@@ -233,9 +233,9 @@
 * API NON-BREAKING: Add 'run.newlineSequences' to schema. https://github.com/oasis-tcs/sarif-spec/issues/169
 * API NON-BREAKING: Add 'rule.deprecatedIds' to schema. https://github.com/oasis-tcs/sarif-spec/issues/293
 * API NON-BREAKING: Add 'versionControlDetails.mappedTo'. https://github.com/oasis-tcs/sarif-spec/issues/248
-* API NON-BREAKING: Add result.rank'. Add 'ruleConfiguration.defaultRank'.
+* API NON-BREAKING: Add result.rank'. Add 'reportingConfiguration.defaultRank'.
```

Comment:
>reportingConfiguration [](start = 43, length = 22)

This should be reverted. Nothing we do in the 01-24 version affects what we published in the 01-09 version.